natsSubjects/src/main/proto/com/arsiu/eduhub/v2/assignmentsvc/output/reqreply/assignment/find_by_id_assignment.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+import "com/arsiu/eduhub/v2/assignmentsvc/commonmodels/assignment/assignment.proto";
+option java_package = "com.arsiu.eduhub.v2.assignmentsvc.output.reqreply.assignment";
+option java_outer_classname = "FindByIdAssignmentResponseProto";
```

Comment:
1. I don't think we need it.
2. Please add proto package([example](https://github.com/OlehSarapuk/ajax-onboarding-project/pull/17/files#diff-7e716f9e29a50261da4ae6372a6a968aaa57c7c695428971ef554d163207de61R3))

Apply to all proto

Reply:
Form what i found this work as expected when protos are in same dir(package) like
  proto 1 `package com.arsiu.eduhub.v2.assignmentsvc.input.reqreply.assignment;`
  proto 2 `package com.arsiu.eduhub.v2.assignmentsvc.input.reqreply.assignment;`
Or do like this when in different packages
```
import "com/arsiu/eduhub/v2/assignmentsvc/commonmodels/assignment/assignment.proto";
package com.arsiu.eduhub.v2.assignmentsvc.input.reqreply.assignment;
option java_outer_classname = "DeleteByIdAssignmentRequestProto";

message DeleteByIdAssignmentRequest {
  com.arsiu.eduhub.v2.assignmentsvc.commonmodels.assignment.AssignmentRequest request = 1;
}
```
Or do as in documentation example
<img width="763" alt="image" src="https://github.com/ArSiu/eduHub/assets/33807305/4e50a972-2101-468f-8cf2-bf5470ccc542">

So I will do and package and leave java_package
```
package com.arsiu.eduhub.v2.assignmentsvc.commonmodels.assignment;
option java_package = "com.arsiu.eduhub.v2.assignmentsvc.commonmodels.assignment";
```

Reply:
Form what i found this work as expected when protos are in same dir(package) like
  ```
proto 1 
      package com.arsiu.eduhub.v2.assignmentsvc.input.reqreply.assignment;
  proto 2 
      package com.arsiu.eduhub.v2.assignmentsvc.input.reqreply.assignment;
```
this works as aspected 
  
But if protos are in different packages it gives an error
`  AssignmentRequest" is not defined.`
  
Or do like this when in different packages
```
import "com/arsiu/eduhub/v2/assignmentsvc/commonmodels/assignment/assignment.proto";
package com.arsiu.eduhub.v2.assignmentsvc.input.reqreply.assignment;
option java_outer_classname = "DeleteByIdAssignmentRequestProto";

message DeleteByIdAssignmentRequest {
  com.arsiu.eduhub.v2.assignmentsvc.commonmodels.assignment.AssignmentRequest request = 1;
}

```
Or do as in documentation example
<img width="467" alt="image" src="https://github.com/ArSiu/eduHub/assets/33807305/093013b7-3a86-40db-b188-bf7dd654492f">

So I came up with this
```
package assignment;

option java_package = "com.arsiu.eduhub.v2.assignmentsvc.commonmodels.assignment";

option java_multiple_files = true;
```

Reply:
also changed 
option java_outer_classname = "UpdateAssignmentResponseProto";
to
option java_multiple_files = true;

Reply:
When protos in different packages then it should be referenced in the correct way. 

You always welcome in private messages(or setup meetings) to discuss it if something is not clear.

Reply:
After more searching came up with this:
commonModels proto
```
syntax = "proto3";

package com.arsiu.eduhub.v2.assignmentsvc.commonmodels.assignment;

option java_multiple_files = true;

...
message AssignmentRequest {
  oneof request {
    AssignmentProto assignment = 1;
    AssignmentId assignment_id = 2;
  }
}
...
```
input reqReply proto
```
syntax = "proto3";

package com.arsiu.eduhub.v2.assignmentsvc.input.reqreply.assignment;

option java_multiple_files = true;

import "com/arsiu/eduhub/v2/assignmentsvc/commonmodels/assignment/assignment.proto";

message DeleteByIdAssignmentRequest {
  commonmodels.assignment.AssignmentRequest request = 1;
}
```